css-two-column-layout/styles.css

```diff
@@ -0,0 +1,72 @@
+body {
+  box-sizing: border-box;
+  margin: 0 auto;
+  font-family: 'Karla', sans-serif, monospace;
+}
+
+.container {
+  width: 800px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  padding-bottom: 20px;
+}
+
+.header {
+  width: 85%;
+  text-align: center;
+  padding: 0 20px;
+  margin: 0 auto;
+}
+
+.header h1 {
+  font-size: 3rem;
+}
+
+.header p {
+  color: rgb(90, 90, 90);
+  font-size: 0.9rem;
+  line-height: 1.8rem;
+}
+
+.column {
+  display: flex;
+}
+
+.text-container {
+  flex-grow: 2;
+}
+
+.logo-container img {
+  width: 100px;
+}
+
+h4 {
+  text-transform: uppercase;
+}
+
+.text-container h4 {
+  width: 60%;
+  padding-top: 10px;
+}
+
+.text-container p {
+  width: 80%;
```

Comment:
I see you setting `width` here, that is an indicator that you likely need a `column` class for each of these elements.